docs/getting-started.md

```diff
@@ -61,16 +69,29 @@ in the next step.
 Once the yarn installation is complete, you can start the app by running the
 following command:
 
+??? Note "Without Yarn 4 Migration"
+
+    If you did not migrate to yarn 4 you can run the following command to start
+    the app.
+
+    ```bash
+    cd backstage
+    yarn install
+    yarn dev
+    ```
+
+Next we start the development environment by running the following command:
 ```bash
-cd backstage
 yarn dev
 ```
 
 This should build and run the app and open a browser window for you. If it does
 not you can open a browser and navigate to `http://localhost:3000` to see the
 app running.
 
```

Comment:
We need to add a note here that tells them that it will open automatically on 127.0.0.1 and you need to use localhost.